book/embeds.md

```diff
@@ -964,7 +964,6 @@ pre-iframe code but wraps the call for the specified `window_id`:
 ``` {.python}
 class JSContext:
     def dispatch_RAF(self, window_id):
```

Comment:
But isn't window_id not defined yet?

Reply:
I think the text above this is also out of place and needs fixing...

Reply:
Fixed PTAL